source/guides/recipes/macos

```diff
@@ -0,0 +1,109 @@
+# Nix(OS) development on macOS
+
+Over recent years, the Nix(OS) community have made great strides in adding support for macOS. Today, it is possible to use Macs for comfortable Nix(OS) development and there is a helpful community in the [`#macos:nixos.org`](https://matrix.to/#/#macos:nixos.org) Matrix channel.
+
+# Nix natively on macOS
+
+First, you need to install Nix on your macOS. Using the [Official Installer](https://nixos.org/download) works fine, but there is no uninstall support. Instead, The [Nix installer from Determinate Systems](https://github.com/DeterminateSystems/nix-installer?tab=readme-ov-file#determinate-nix-installer) is the recommended installer for Nix on macOS, especially if you plan on using flakes. If you need help, follow https://nixcademy.com/posts/nix-on-macos/.
```

Comment:
Please don't use passive on subjective claims. Recommended by whom? I understand it's `nix-darwin` maintainers. Maybe it would help readers to orient themselves if we started the article with an overview of the situation and the general approach taken here.